init.lua

```diff
@@ -1,15 +1,38 @@
+local function freeze(player)
```

Comment:
You could also update `default.player_attached[name]` here. Both are always used together.